src/Logging/src/DynamicLogger/DynamicConsoleLoggerProvider.cs

```diff
@@ -27,11 +27,16 @@ public class DynamicConsoleLoggerProvider : DynamicLoggerProviderBase
     /// message processors to apply to message.
     /// </param>
     public DynamicConsoleLoggerProvider(IOptionsMonitor<LoggerFilterOptions> filterOptions, ConsoleLoggerProvider consoleLoggerProvider,
-        IEnumerable<IDynamicMessageProcessor> messageProcessors = null)
+        IEnumerable<IDynamicMessageProcessor> messageProcessors)
         : base(() => consoleLoggerProvider, GetInitialLevelsFromOptions(filterOptions), messageProcessors)
     {
     }
 
+    public DynamicConsoleLoggerProvider(IOptionsMonitor<LoggerFilterOptions> filterOptions, ConsoleLoggerProvider consoleLoggerProvider)
```

Comment:
Is there a case where these constructors are used in applications? If so, we should add the XML comments from the other constructor and (really minor nit) this code would probably read a little more naturally if the constructors are reversed in order. If we don't think these will be used downstream, should they still be public?

Reply:
I wasn't looking at the Public API for Steeltoe.Logging atm, but I went ahead with the search and replace of null parameters. Should we add Steeltoe.Logging to the Public api analyzers. About this particular type, I think it can be internal following our other changes to lock down the API.

Reply:
Ok, that is a fair point. I could go either way on whether the broader logging review is included in this PR, I'll let you decide on that. We do need to get to it done eventually either way